documentation/modules/ROOT/pages/configuration/avro.adoc

```diff
@@ -49,10 +49,15 @@ To use Apache Avro serialization, you must deploy a schema registry that manages
 For information about setting up this registry, see the documentation for link:{LinkServiceRegistryInstall}[{NameServiceRegistryInstall}].
 endif::product[]
 
+// Type: concept
+// Title: {registry}
+[id="apicurio-registry"]
+== {registry-name-full}
+
 // Type: concept
 // Title: About the {registry}
 [id="about-the-registry"]
-== About the {registry-name-full}
+=== About the {registry-name-full}
```

Comment:
I don't understand the logic behind adding the redundant title heading here, as it doesn't really add anything.  I suggest eliminating lines 52-56 in favor of lines 57-60, and restoring the second level heading level on L60.

Reply:
There are two independent schema registry - Apicurio and Confluent. Currently sub-chapters regarding Apicurio and Confluent are on the same level without any separation, which can be harder to read and understand, making the feel that they are the same thing, which is not true. This adds higher level chapter `Apicurio API and Schema Registry` and later on this PR also adds `Confluent Schema Registry` chapter which separates these two technologies into dedicated sections, logically separating these two independent technologies. So instead of current

* About the Apicurio API and Schema Registry
* Deployment overview
* Deploying with Debezium containers
* Confluent Schema Registry
* Naming
* Getting More Information

the structure now is as follows:

* Apicurio API and Schema Registry
  * About the Apicurio API and Schema Registry
  * Apicurio Registry deployment overview
  * Deploying Apicurio Registry with Debezium containers
* Confluent Schema Registry
  * Confluent Schema Registry deployment overview
  * Deploying Confluent Schema Registry with Debezium containers
* Naming
* Getting More Information